### DISCUSSION TITLE
Help please?

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#13948] [Q&A] [answered]
Can anyone help fix the following please?

`pkg install proot-distro
Checking availability of current mirror:
[*] https://mirrors.sdu.edu.cn/termux/termux-main: ok
Hit:1 https://mirrors.sdu.edu.cn/termux/termux-main stable InRelease
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 ncurses-utils : Depends: ncurses (= 6.3-2) but 6.3-4 is to be installed
E: Unable to correct problems, you have held broken packages.`

**Comment by macerg** [comment, marked as answer]
Problem gone away.

**Comment by xtkoba** [comment]
I believe this is due to #13977 (mirror selection of `pkg` command being broken.)

I would recommend users to stop using `pkg` command and to use commands from `dpkg` suite directly (e.g. `apt-get`.)

**Reply by macerg** [reply]
Ok, thanks. I do normally use 'apt install'. Don't know why I used 'pkg' on this occasion. Ta.
